### PR TITLE
Announcements - Week 33

### PR DESCRIPTION
**Pull request by sslavic**

### CODE REVIEW

File: announcements/2020-Week-33/apps.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+**Optional Apps Releases**
+(Can be installed from App Catalogs)
+
+1. [NGINX IC v1.9.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#190---2020-08-13) adds support for Ingress resources validating webhook.
+
+2. [NGINX IC v1.9.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#191---2020-08-14) configures explicit helm hook weights to make validating webhook resource management reliable.
+
+(Note: NGINX is Optional and not Pre-Installed as of AWS 10+, Azure 12+, and KVM 12.2+ platform releases.)
+
+3. [Strimzi Kafka Operator](https://github.com/giantswarm/strimzi-kafka-operator-app) is now available as a Playground Catalog app and can be used to test and play with Kafka app on tenant clusters.
```
Comment (cokiengchiara) [inline announcements/2020-Week-33/apps.md:12]: @sslavic does this look fine to you?

File: announcements/2020-Week-33/apps.md [hunk 2]
```diff
@@ -7,7 +7,9 @@
 
 2. [NGINX IC v1.9.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#191---2020-08-14) configures explicit helm hook weights to make validating webhook resource management reliable.
 
-(Note: NGINX is Optional and not Pre-Installed as of AWS 10+, Azure 12+ and KVM 12.2+ platform releases)
+(Note: NGINX is Optional and not Pre-Installed as of AWS 10+, Azure 12+, and KVM 12.2+ platform releases.)
+
+3. [Strimzi Kafka Operator](https://github.com/giantswarm/strimzi-kafka-operator-app) is now available as a Playground Catalog app and can be used to test and play with Kafka app on tenant clusters.
```
Comment (sslavic) [inline announcements/2020-Week-33/apps.md:12]:
```suggestion
3. [Strimzi Kafka Operator](https://github.com/giantswarm/strimzi-kafka-operator-app) is now available as a Playground Catalog app and can be used to test and play with Apache Kafka clusters.
```

File: announcements/2020-Week-33/apps.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# :zap: This Week in Non-Platform Releases (August 14, 2020):zap:
```
Comment (cokiengchiara) [inline announcements/2020-Week-33/apps.md:1]: @giantswarm/sig-product and maybe @stone-z :D

We're sending non-provider release notes (Optional Apps, Web Interface, Control Plane), etc. as a separate file and announcements. This reduce noise and duplicate content in release announcements. Not sure if you have better ideas on what to call this aside from "Non-Platform" 
* Other Releases?
* Non-Provider Releases?
* Provider Independent?

Comment (cokiengchiara) [inline announcements/2020-Week-33/apps.md:1]:
```suggestion
# :zap: This Week in Provider-Independent Releases (August 14, 2020):zap:
```

**Pull request merged**